### ISSUE TITLE
Where is the button to add a secondary configuration?

### ISSUE THREAD

**Issue by N247S** [KoenZomers/KeePassOneDriveSync#70]
I noticed that there is no way to add a secondary configuration for a singel file (or just to add a configuration without base file). So either there is no way to achieve that (eventhough it should be possible following the information), or I am missing something really obvious

**Comment by KoenZomers** [comment]
Correct. It is currently based on a 1:1 comparison between a KeePass database and a location where to sync it to. Is your scenario that you would like to sync the same file to multiple OneDrives at the same time?

**Comment by KoenZomers** [comment]
Ok thanks for elaborating @N247S, I'll consider adding it.

**Comment by sebrock** [comment]
> Correct. It is currently based on a 1:1 relationship between a KeePass database and a location where to sync it to. Is your scenario that you would like to sync the same file to multiple OneDrives at the same time?

I would like to sync to two different OneDrives (one business and one personal), and keep them consistent.

**Comment by KoenZomers** [comment]
Going to try to clean ship now I have reopened this repository. As this is still an old issue, I will close it for now. If it would still be relevant to you, feel free to post back and we can reopen and look into it.
